packages/store-sync/src/postgres-decoded/buildColumn.ts

```diff
@@ -234,7 +243,7 @@ export function buildColumn(name: string, schemaAbiType: SchemaAbiType) {
     case "bytes31[]":
     case "bytes32[]":
     case "bool[]":
-      return asJson(name);
+      return asBoolArray(name);
```

Comment:
erm, I think this applying the column type to all the above array types, not _just_ the bool type

Reply:
since this is a column change, should bump the postgres schema/sync version

Reply:
Oh thanks - been looking at these column types for too long 😵‍💫